src/obstacle_avoidance/launch/obstacle_avoidance.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+    <node name="my_node" pkg="sample_package" type="my_node" output="screen">
+        <param name="character" value="?" />
+    </node>
+</launch>
```

Comment:
This file is launching the sample node. It should either be revised to launch the obstacle avoidance node, or just deleted.